src/locales/en/translation.json

```diff
@@ -2715,36 +2715,9 @@
     }
   },
   "promotions": {
-    "card1": {
-      "title": "5K SOV weekly rewards",
-      "duration": "Ongoing weekly rewards",
-      "text": "Provide a 1:1 ratio of MYNT and RBTC to the MYNT/BTC AMM liquidity pool and instantly start accruing your share of 5,000 SOV rewards."
-    },
-    "card2": {
-      "title": "2.5K SOV weekly rewards",
-      "duration": "Ongoing weekly rewards",
-      "text": "Provide any amount of XUSD to the XUSD lending pool and instantly start accruing your share of 2,500 SOV rewards."
-    },
-    "card3": {
-      "title": "5K SOV weekly rewards",
-      "duration": "Ongoing weekly rewards",
-      "text": "Provide a 1:1 ratio of XUSD and RBTC to the XUSD/RBTC AMM liquidity pool and instantly start accruing your share of 5000 SOV rewards."
-    },
-    "card4": {
-      "title": "10K SOV weekly rewards",
-      "duration": "Ongoing weekly rewards",
-      "text": "Provide a 1:1 ratio of SOV and RBTC to the SOV/RBTC AMM liquidity pool and instantly start accruing your share of 10,000 SOV rewards."
-    },
-    "card5": {
-      "title": "5K SOV weekly rewards",
-      "duration": "Ongoing weekly rewards",
-      "text": "Provide a 1:1 ratio of ETH and RBTC to the ETH/RBTC AMM liquidity pool and instantly start accruing your share of 5,000 SOV rewards."
-    },
-    "card6": {
-      "title": "20K SOV weekly rewards",
-      "duration": "Ongoing weekly rewards",
-      "text": "Provide a 1:1 ratio of DLLR and RBTC to the DLLR/RBTC AMM liquidity pool and instantly start accruing your share of 20,000 SOV rewards."
-    },
+    "title": "{{rewardAmount}} SOV weekly rewards",
+    "duration": "Ongoing weekly rewards",
+    "text": "Provide a 1:1 ratio of {{asset1}} and {{asset2}} to the {{asset1}}/{{asset2}} AMM liquidity pool and instantly start accruing your share of {{rewardAmount}} SOV rewards.",
```

Comment:
Update the same in other localisation files (remove updated props if needed, translators will add them back later with new localized copy)

Reply:
Update, can be re-reviewed.